### PR TITLE
added redirect to particular FAQ to components

### LINKED ISSUE #373: Opening up Smart-faq in kiwicom does not have animation
Old Help does have animation.

### LINKED ISSUE #383: Show previously displayed booking
Acceptance criteria

- When I open SmartFAQ anywhere on the frontend, it shows the nearest booking
- When I open SmartFAQ in MMB in booking detail page, this specific booking is opened
- When I open SmartFAQ anywhere, MMB booking detail included, and select different booking from the allbookings page, then I close SmartFAQ and reopen it again no matter on which page of the frontend, SmartFAQ is smart enough to display booking previously selected by a user 

=> in other words, if the user intentionally selected one booking from the list, keep track of such selection so it opens the same booking later again, but don't implement such memoization for specific booking which was displayed just because the user was on MMB detail page, makes sense?

### PR DESCRIPTION
**Pull request by dinodsaurus**
closes #383 
closes #373 

First we need to merge https://gitlab.skypicker.com/frontend/frontend/merge_requests/5207

We cannot use `initial-route` because `smart-faq` component is initialized on startup and `initial-route` is `/` so we need to check it on each re-renedr`<br/><br/><br/><url>LiveURL: https://smartfaq-refactor-initial-route.surge.sh</url>

### CODE REVIEW

**Comment by Strajk** [comment]
Does this really close #373? @dinodsaurus 
I don't see any changes related to animations in diff

**Comment by dinodsaurus** [comment]
@Strajk sorry I forgot to reference this issue to gitlab.. All fixed regarding animation and some other refactoring stuff can be seen there. https://gitlab.skypicker.com/frontend/frontend/merge_requests/5207

File: src/common/RedirectToFaq.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+const RedirectsLoggedIn = (props: Props) => {
+  return (
+    <React.Fragment>
+      <Redirector initialRoute={props.initialRoute} />
+      {props.children}
+    </React.Fragment>
+  );
+};
+
+export default RedirectsLoggedIn;
```
Comment (jaroslav-kubicek) [inline src/common/RedirectToFaq.js]: exported name doesn't match with the filename

File: src/Routes.js [hunk 2]
```diff
@@ -16,18 +16,26 @@ type Props = {|
 
 const Routes = (props: Props) => {
   return (
-    <MemoryRouter initialEntries={[props.initialRoute]} initialIndex={0}>
+    <MemoryRouter initialEntries={['/']} initialIndex={0}>
```
Comment (jaroslav-kubicek) [inline src/Routes.js]: I guess `initialEntries={['/']} initialIndex={0}` not needed anymore?

**Comment by dinodsaurus** [comment]
I close this in favour of #478

**Pull request closed without merging**